src/__tests__/npm.test.js

```diff
@@ -117,7 +117,7 @@ describe('getDownloads()', () => {
     );
 
     // eslint-disable-next-line no-console
-    console.log('downloads', { jest, angular, holmes });
+    // console.log('downloads', { jest, angular, holmes });
```

Comment:
```suggestion
    console.log('downloads', { jest, angular, holmes });
```

(easier to update like this when the test get outdated)